users/stdiscosrv.rst

```diff
@@ -68,6 +68,10 @@ list, e.g. ``https://disco.example.com:8443/``. Note that stdiscosrv uses port
 8443 by default. For stdiscosrv to be available over the internet with a dynamic
 IP address, you will need a dynamic DNS service.
 
+.. warning::
+   Versions prior to ``0.14.44`` may need ``/v2/`` appended to the discovery
+   server address, e.g. ``https://disco.example.com:8443/v2/``.
```

Comment:
I'd use `.. deprecated:: v0.14.44` and remove the may, something like 
> Prior versions needed ``v2/`` appended to the discovery  server address, e.g. ``https://disco.example.com:8443/v2/``."

Reply:
I think depending on you version (for example if it's even older) it may no be needed.